src/styles/app.scss

```diff
@@ -19,6 +19,7 @@
   padding: 0rem 0rem;
   padding-right: .5rem !important;
   padding-left: .5rem !important;
+  font-weight: 700;
```

Comment:
@Duncanian, I hope you realized that the logo or anchor tag with a class of `navbar-brand` had strong tags but has no class of `nav-link`.

Reply:
Let me add it ASAP

Reply:
Done.